app/src/main/java/com/eveningoutpost/dexdrip/Models/BgReading.java

```diff
@@ -709,9 +710,9 @@ public void find_new_curve() {
         }
     }
 
-    public void calculateAgeAdjustedRawValue(){
+    public void calculateAgeAdjustedRawValue(Context context){
```

Comment:
please start the function with:
if(CollectionServiceStarter.isLimitter(context)) {
    age_adjusted_raw_value = raw_data;
    return;
}

it will make it much readable.